### PR TITLE
QoL bug fixes

### PR DESCRIPTION
**Pull request by mgchandler**
Fixes to some errors which flag up.
Namely:
- Setting aspect in plotting
- New versions of BRAIN no longer save to ph_velocity
- Use of safe_load in yaml
Also added:
- Hanning filter, useful in my own work.

### CODE REVIEW

**Review by nbud — COMMENTED** [review]
Thank you for the pull request Matt, overall it looks very good! Regarding the replacement of `dtype=float`: it was much needed indeed, but for the record I still get many deprecation warnings caused by the same issue in other parts of the code when I run pytest (pytest 7.3 and numpy 1.23), maybe we could fix that in the future.

File: arim/signal.py [hunk 1]
```diff
@@ -176,6 +177,62 @@ def __call__(self, arr, axis=-1, **kwargs):
...
+        peak_pos_fract = centre_freq / max_freq
+        half_width_fract = half_bandwidth / max_freq
+        r = np.arange(nsamples) / (nsamples - 1)
+        r1 = 0.5 * (1 + np.cos((r - peak_pos_fract) / half_width_fract * np.pi))
+        self.samples = nsamples
+        self.centre_freq = centre_freq
+        self.half_bandwidth = half_bandwidth
+        self.max_freq = max_freq
+        self.filter_window = r1 * np.logical_and((r >= (peak_pos_fract - half_width_fract)), (r <= peak_pos_fract + half_width_fract))
+        # print('Hanning')
```
[the remark this answers is not in the document]
Comment (mgchandler) [reply arim/signal.py:221]: This is fair, I implemented `Hanning` as a direct copy of `Gaussian` with relevant changes.

File: tests/data/brain/exp_data.mat7.mat
[the remark this answers is not in the document]
Comment (mgchandler) [reply tests/data/brain/exp_data.mat7.mat:1]: BRAIN stores wave speed inside `exp_data.material.vel_spherical_harmonic_coeffs`, I updated this file to store speed here.

File: tests/im/test_geomed.py [hunk 2]
```diff
@@ -28,7 +28,7 @@ def test_gradf_and_inv_hessf():
     np.testing.assert_allclose(h22, hessref[1, 1], **tol)
 
 
-@pytest.mark.parametrize("dtype", [np.float32, np.float64])
+@pytest.mark.parametrize("dtype", [float])
```
[the remark this answers is not in the document]
Comment (mgchandler) [reply tests/im/test_geomed.py:31]: Yes I had conflated the three numpy floats with one another. Happy to revert.

File: tests/data/brain/exp_data.mat7.mat
[the remark this answers is not in the document]
Comment (nbud) [reply tests/data/brain/exp_data.mat7.mat:1]: Ah I see. It's fine as it is now, but if we ever make larger changes in the reader, we could keep multiple versions of the same data file, and ensure with a unit test that arim can read all versions.

**Comment by nbud** [comment]
Looks good, thank you!

**Pull request merged**